### ISSUE TITLE
Add endpoint for users' home feed

### ISSUE THREAD

**Issue by silverAndroid** [BrandonDanis/FriendStagram-Backend#4]
This is different from /users/:username because that is just the posts that a specific user has made. This endpoint would be a combination of all your friends' posts ordered in some way (whether it be chronologically or by popularity).

**Comment by BrandonDanis** [comment]
Yeah, the /users/:username is used when visiting a profile. We need to make another endpoint such as /feed that would deliver the x most recent/popular posts from the users you follow. 

That being said, we need to create another task that add the follow feature.
